### ISSUE TITLE
Use other components as children

### ISSUE THREAD

**Issue by andyearnshaw** [blueberryapps/react-bluekit#109]
If I set `propTypes.children` to `PropTypes.node.isRequired`, I should be able to type a child component into the `children*` input.  For example, let's say I have a `TabStrip` and `TabPanel` component that work together like this:

```jsx
<TabStrip>
  <TabPanel link="Tab 1">
     ...tab content
  </TabPanel>
</TabStrip>
```

This is difficult to preview in BlueKit because `<TabPanel>` isn't accessible when entered into the `children*` input.

**Comment by ondrejbartas** [comment]
Yep that is right and there is no way to achieve this. At least I don't see how. What we are doing is that we create Example.react.js with this large nested structures and this is what we see in bluekit. Into production build these examples will not propagate, unless you use them.
